atomic_reactor/inner.py

```diff
@@ -452,6 +513,7 @@ def build_docker_image(self):
             finally:
                 self.source.remove_tmpdir()
 
+            self.fs_watcher.finish()
```

Comment:
Should this go under the `finally` block above? If an exit plugin fails, this line is not reached.

Reply:
That would be unlikely to cause much concern :) but sure, to be tidy.